app/handlers/configHomography.py

```diff
@@ -10,17 +9,15 @@
 from ast import literal_eval
 from baseHandler import BaseHandler
 
-class UploadHomographyHandler(BaseHandler):
+class ConfigHomographyHandler(BaseHandler):
     """
-    @api {post} /uploadHomography/ Upload Homography
-    @apiName UploadHomography
-    @apiVersion 0.1.0
+    @api {post} /configHomography/ Config Homography
+    @apiName ConfigHomography
+    @apiVersion 0.2.0
     @apiGroup Upload
```

Comment:
Should this still be Upload? Should upload and config be merged? Should we make it clearer that this needs to be done.